interface/patient_file/summary/add_edit_issue.php

```diff
@@ -723,6 +723,15 @@ function divclick(cb, divid) {
   </td>
  </tr>
 
+ <tr id='row_returndate'>
+   <td>
+   <input type='hidden'  name='form_return' id='form_return' />
+   <input type='hidden'  name='row_reinjury_id' id='row_reinjury_id' />
+  <img  
+    id='img_return'/>
+  </td>
+ </tr>
+ 
```

Comment:
this is a fix that was put in by someone and it works but it will need to be investigated down the road